[1m==================================== test session starts =====================================
platform linux -- Python 3.9.17, pytest-6.2.2, py-1.11.0, pluggy-0.13.1 -- /opt/anaconda3/envs/mlflow-fce2acde6a44ec774359058ced033a5e8981989c/bin/python
cachedir: .pytest_cache
rootdir: /home/aditya/genre_classification/check_data
[1mcollected 4 items                                                                            
test_data.py::test_column_presence_and_type [32mPASSED
test_data.py::test_class_names [32mPASSED
test_data.py::test_column_ranges [32mPASSED
test_data.py::test_kolmogorov_smirnov [32mPASSED
[33m====================================== warnings summary ======================================
test_data.py::test_column_presence_and_type
  /opt/anaconda3/envs/mlflow-fce2acde6a44ec774359058ced033a5e8981989c/lib/python3.9/site-packages/pluggy/callers.py:187: DtypeWarning: Columns (16) have mixed types.Specify dtype option on import or set low_memory=False.
    res = hook_impl.function(*args)
-- Docs: https://docs.pytest.org/en/stable/warnings.html
[33m=============================== [32m4 passed[39m, [33m[1m1 warning[22m in 13.47s ================================